include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -6356,7 +6359,7 @@ namespace VectorTools
 
       // append result of this cell to the end of the vector
       AssertIsFinite(diff);
-      return diff;
+      return diff; // TODO: what about diff_mean? Not necessarily double
```

Comment:
i actually don't know ;-) the point is that the norm should be real-valued. In that sense, the mean-value is not a norm, actually. I guess, i will add an assert and return real-part of the mean value for complex-valued vectors. Does this sound ok to you?